frontend/src/features/admin-form/responses/ResponsesPage/storage/UnlockedResponses/ResponsesTable/SendReminderButton.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react'
+import { BiBell, BiCheck } from 'react-icons/bi'
+import { useParams } from 'react-router-dom'
+import { Text } from '@chakra-ui/react'
+
+import Button from '~components/Button'
+
+import { useFormRemindersMutations } from '~features/admin-form/common/mutations'
+import { useGetIndividualDecryptedSubmission } from '~features/admin-form/responses/IndividualResponsePage/queries'
+
+export const SendReminderButton = ({
+  submissionId,
+}: {
+  submissionId: string
+}) => {
+  const { formId = '' } = useParams()
+
+  const { sendReminderForResponseMutation } = useFormRemindersMutations()
+
+  const sendReminderForResponse = sendReminderForResponseMutation
+
+  const [isSent, setIsSent] = useState(false)
+
+  const { data: submissionData, isLoading: isLoadingSubmissionData } =
+    useGetIndividualDecryptedSubmission({
+      formId,
+      submissionId,
+    })
+
+  // Used to define a test key for Storybook UI Testing
+  const isTest = import.meta.env.STORYBOOK_NODE_ENV === 'test'
+  const submissionSecretKey = isTest
+    ? 'storybook-test-dummy-key'
```

Comment:
This is used as a workaround for chromatic tests as mocking the `useGetIndividualDecryptedSubmission` is rather complicated -- requiring a valid key-pair and decrypt-able payload for mrf.

Reply:
removed this